### PR TITLE
Created CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by anshulsoni4**
Please check the contributing.md file that i have added 
let me know if any changes are required 

Issue - #5

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# Contributing Guidelines 👨‍💻
+## 👨‍💻 Prerequisite Skills to Contribute
+
+### 💻 Contribute in public/Profile
+
+- [Git](https://git-scm.com/) 
+- [Basic JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+- [ReactJS](https://reactjs.org/docs/getting-started.html)
```
Comment (SGI-CAPP-AT2) [inline CONTRIBUTING.md]: No need of react js

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
+# Contributing Guidelines 👨‍💻
+## 👨‍💻 Prerequisite Skills to Contribute
+
+### 💻 Contribute in public/Profile
```
Comment (SGI-CAPP-AT2) [inline CONTRIBUTING.md]: Replace `public/Profile` with `tool/`

Add html & hljs as a prerequisite

**Review by SGI-CAPP-AT2 — COMMENTED** [review]
Check review 
& Create Changes accordingly

_shgi_

**Comment by anshulsoni4** [comment]
@SGI-CAPP-AT2 pls review now

**Comment by SGI-CAPP-AT2** [comment]
Done !

**Comment by anshulsoni4** [comment]
@SGI-CAPP-AT2 Thankyou so much 😄

**Comment by SGI-CAPP-AT2** [comment]
@all-contributors please add @anshulsoni4 for infrastructure, tests and code

**Comment by anshulsoni4** [comment]
@SGI-CAPP-AT2 thanks

**Pull request merged**